examples/trpo_bullet_kuka.py

```diff
@@ -0,0 +1,57 @@
+from rllab.algos.trpo import TRPO
+from rllab.baselines.linear_feature_baseline import LinearFeatureBaseline
+from rllab.envs.bullet_env import BulletEnv
+from rllab.envs.normalized_env import normalize
+from rllab.misc.instrument import run_experiment_lite
+from rllab.policies.gaussian_mlp_policy import GaussianMLPPolicy
+
+from rllab.misc.rllab_gym_register import Register
+
+
+def run_task(*_):
+    # Please note that different environments with different action spaces may require different
+    # policies. For example with a Box action space, a GaussianMLPPolicy works, but for a Discrete
+    # action space may need to use a CategoricalMLPPolicy (see the trpo_gym_cartpole.py example)
+
+    #Register the env in gym registry and get the id
+
+    r = Register("pybullet_envs.bullet", "KukaGymEnv", max_episode_steps=1000, reward_threshold=1.0, params = {"renders":True})
```

Comment:
We need this to be completely transparent to the user, which means manual registration is not an option.

It needs to be something like
```python
from rllab.envs.bullet_env import BulletEnv

env = normalize(BulletEnv("KukaGymEnv"))

Reply:
There are a few inbuilt environments in pybullet, `KukaGymEnv` is one of them. 
I can change the code to this.

`env = normalize(BulletEnv("KukaGymEnv"))`

But, if any new environment has to be added, which does not exist in `pybullet` or `gym`, then the only way is to manually register the environment because where would we get the `max_episode_steps` and `reward_threshold` from? These are inputs specific to the environment stored in `EnvSpec` class by gym (which is initialized by the register method) right?
How would it work then?

Reply:
Just to clarify, `"Furthermore, we'd like to use other unofficial Gym environments in rllab, but it is currently difficult for the same reason."` means that we would like to write our own env which extends `gym.Env` and use those right?

If so, how would I set the `max_episode_steps`, `reward_threshold` parameters?

Reply:
https://github.com/openai/gym/blob/master/gym/envs/registration.py#L25

You just have to call register at some point . In Python, people usually do things like this automatically by adding function calls to `mypackage/__init__.py`, which is executed on package import. In fact, pybullet already does this.
https://github.com/bulletphysics/bullet3/blob/master/examples/pybullet/gym/pybullet_envs/__init__.py

So it seems to me that as long as you import `pybullet.gym.pybullet_envs` somewhere in your implementation, you should be good to go.